addon/-legacy-private/system/references/belongs-to.js

```diff
@@ -398,8 +398,9 @@ export default class BelongsToReference extends Reference {
      @method reload
      @return {Promise} a promise that resolves with the record in this belongs-to relationship after the reload has completed.
   */
-  reload() {
-    return this.belongsToRelationship.reload().then(internalModel => {
+  reload(options = {}) {
+    options.reload = true;
```

Comment:
we can delete this assignment

Reply:
We can also delete the `options = {}` assignment since we no longer care about ensuring options is defined.